docs/requirements.txt

```diff
@@ -25,3 +25,4 @@ torchvision==0.14.1
 torchaudio==0.13.1
 timm==0.6.12
 datasets>=2.9.0
+xgboost==1.7.2
```

Comment:
confirm this is actually required?

Reply:
This was used by Mayank in an old version of the tutorial but not anymore, removed!